exporters/otlp/common/build.gradle.kts

```diff
@@ -11,17 +11,24 @@ otelJava.moduleName.set("io.opentelemetry.exporter.otlp.internal")
 
 dependencies {
   api(project(":api:all"))
-  api(project(":proto"))
   api(project(":sdk:all"))
   api(project(":sdk:metrics"))
 
+  // We only use the protos for HTTP, logging-otlp, and metric export for now. Let them expose the
+  // proto dependency in their POMs instead of here. This is fine because this artifact only
+  // contains internal code.
+  compileOnly(project(":proto"))
+
   implementation("com.google.protobuf:protobuf-java")
 
+  // Similar to above note about :proto, we include a helper shared by the gRPC exporters but do not
```

Comment:
I suppose what we really need for these is `exporter-grpc-common`, which would also be used from the jaeger exporter. It is yet another artifact, but "makes sense" and would allow Jaeger TLS support to be functioning (presumably no one uses it with TLS right now since without customizing certs it's rare that it could function)